### ISSUE TITLE
Only 1 geojson layer is rendered when multiple layers are used

### ISSUE THREAD

**Issue by kygoh** [kygoh/lflt-core#6]
The following template will only render either `layer1$` or `layer2$`:

```html
<lflt-map class="map-container">
  <lflt-tile-layer [urlTemplate]="tileLayerUrl" [options]="tileLayerOptions"></lflt-tile-layer>

  <lflt-geojson-layer [geojson]="layer1$ | async" [options]="geojsonOptions">
  </lflt-geojson-layer>

  <lflt-geojson-layer [geojson]="layer2$ | async" [options]="geojsonOptions">
  </lflt-geojson-layer>
</lflt-map>
```

Supposedly, both geojson layers should be rendered.

**Comment by kygoh** [comment]
L.GeoJSON layer will be added to the map in

https://github.com/kygoh/lflt-core/blob/ed47e10dcbbe14ee88a38d9cece3fcb96a1ab9ae/projects/lflt-core/src/lib/directives/leaflet-layer.directive.ts#L74-L78

The `layer` in line 75 is instantiated in

https://github.com/kygoh/lflt-core/blob/ed47e10dcbbe14ee88a38d9cece3fcb96a1ab9ae/projects/lflt-core/src/lib/directives/leaflet-layer.directive.ts#L134-L138

It appears that `this.geojson` is empty for the layer that was not rendered.

**Comment by kygoh** [comment]
> It appears that `this.geojson` is empty for the layer that was not rendered.

Looks like either one of `[geojson]="layer1$ | async"` or `[geojson]="layer2$ | async"` completed first and broadcasted `LFLT_GEOJSON_LAYER_READY` through

https://github.com/kygoh/lflt-core/blob/ed47e10dcbbe14ee88a38d9cece3fcb96a1ab9ae/projects/lflt-core/src/lib/directives/leaflet-layer.directive.ts#L126-L132

The broadcasted event will be picked up using

https://github.com/kygoh/lflt-core/blob/ed47e10dcbbe14ee88a38d9cece3fcb96a1ab9ae/projects/lflt-core/src/lib/directives/leaflet-layer.directive.ts#L38-L42

through the following subscription, which in turn triggered `whenReady`

https://github.com/kygoh/lflt-core/blob/ed47e10dcbbe14ee88a38d9cece3fcb96a1ab9ae/projects/lflt-core/src/lib/directives/leaflet-layer.directive.ts#L44-L47

However,  `whenReady` in both, in fact all, `LeafletGeoJSONLayerDirective`s were triggered when only one is ready to `initComponent`:

https://github.com/kygoh/lflt-core/blob/ed47e10dcbbe14ee88a38d9cece3fcb96a1ab9ae/projects/lflt-core/src/lib/directives/leaflet-layer.directive.ts#L70-L72

**Comment by kygoh** [comment]
Fixed by commit 81a9be2
